app/build.gradle

```diff
@@ -13,6 +13,9 @@ def isRunningOnTravisAndIsNotPRBuild = System.getenv("CI") == "true" && file('..
 if(isRunningOnTravisAndIsNotPRBuild) {
     apply plugin: 'com.github.triplet.play'
 }
+repositories {
+    mavenCentral()
+}
```

Comment:
I believe this isn't required. We already have the `mavenCentral()` repository in the top-level `build.gradle`.